src/main/java/gregtech/api/unification/material/type/SolidMaterial.java

```diff
@@ -32,6 +32,7 @@
         public static final long GENERATE_LONG_ROD = createFlag(22);
         public static final long MORTAR_GRINDABLE = createFlag(24);
         public static final long GENERATE_FRAME = createFlag(45);
+        public static final long GENERATE_METAL_CASING = createFlag(65);
```

Comment:
that flag should include GENERATE_FRAME | GENERATE_PLATE automatically. See how it is implemented for other things like long sticks, which generate sticks when being added in solo.